### PR TITLE
moved login button

### PR DESCRIPTION
**Pull request by sebastianovide**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -2,25 +2,108 @@ import React from 'react';
 
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
-// import IconButton from '@material-ui/core/IconButton';
-// import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+
+
```
Comment (joykuotw) [inline src/components/Header.js]: redundant lines

File: src/components/Header.js [hunk 2]
```diff
@@ -2,25 +2,108 @@ import React from 'react';
...
+  handleClose = () => {
+    this.setState({ anchorEl: null });
+  };
+
+  render() {
+    const open = Boolean(this.state.anchorEl);
+
+    return <div className="geovation-header">
+      <AppBar position="static" className="header">
```
Comment (joykuotw) [inline src/components/Header.js]: there is no header class.

File: src/components/Header.js [hunk 3]
```diff
@@ -2,25 +2,108 @@ import React from 'react';
...
+            <MenuIcon />
+          </IconButton>
+
+          <Typography variant="h6" color="inherit" className="grow">
+            {this.props.headline}
+          </Typography>
+
+          <div>
+            <IconButton
+              // aria-owns={open ? 'menu-appbar' : undefined}
```
Comment (joykuotw) [inline src/components/Header.js]: why comment this out?

File: src/components/Header.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.geovation-header {
+  flex-grow: 1;
+
+  .grow {
+    flex-grow: 1;
+  }
+
+  .menuButton {
+    margin-left: -12;
```
Comment (joykuotw) [inline src/components/Header.scss]:
```suggestion
    margin-left: -12px;
```

File: src/components/Header.scss [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    margin-right: 20;
```
Comment (joykuotw) [inline src/components/Header.scss]:
```suggestion
    margin-right: 20px;
```

File: src/components/Header.scss (hunk 4 above)
Comment (joykuotw) [reply src/components/Header.scss]: need to have unit or will be invalid property value

File: src/components/Header.scss (hunk 5 above)
Comment (joykuotw) [reply src/components/Header.scss]: same as above

File: src/components/Header.js (hunk 3 above)
Comment (sebastianovide) [reply src/components/Header.js]: you will use it if need to open the menu

File: src/components/Header.js (hunk 1 above)
Comment (sebastianovide) [reply src/components/Header.js]: done

File: src/components/Header.js (hunk 2 above)
Comment (sebastianovide) [reply src/components/Header.js]: done

**Comment by joykuotw** [comment]
@sebastianovide might need to rebase to the master first, or the CI will never pass

**Pull request merged**